pkg/controllers/managementuser/rbac/prtb_handler_test.go

```diff
@@ -186,3 +189,475 @@ func Test_manager_checkForGlobalResourceRules(t *testing.T) {
 		assert.Equal(t, test.want, got, fmt.Sprintf("test %v failed", test.name))
 	}
 }
+
+func Test_ensureProjectResourcesRoleBindings(t *testing.T) {
+	var roleBindings map[string]map[string]*rbacv1.RoleBinding
+	tests := []struct {
+		name    string
+		binding *v3.ProjectRoleTemplateBinding
+		cache   map[string]map[string]*rbacv1.RoleBinding
+		want    *rbacv1.RoleBinding
+		wantErr error
```

Comment:
I like checking the error string because I've seen error messages that use the wrong variables or forget the interpolation variables so the message ends up with `%!s(MISSING)` in it.

Reply:
Fair enough, that makes sense.